.github/workflows/check_release_version.yml

```diff
@@ -4,24 +4,24 @@ name: Check release version
 on:
   pull_request:
     branches:
-      - main
+      - master
```

Comment:
This branch name shouldn't be here.  The main branch is called `main` in this repo.